my/fillet.scad

```diff
@@ -40,9 +40,9 @@ module offset_3d(r=1.0) {
   }
 }
 
-/* example:
+// example:
```

Comment:
This example was commented out on purpose. This fillet is used as submodule through `#include` directives, so this breaks those models.